src/SpaceTime/angular/AngularVelocity.cs

```diff
@@ -48,47 +39,36 @@ public static AngularVelocity FromDegrees(float degrees)
         /// <summary>
         /// Returns the numeric value of the angular velocity in radians.
         /// </summary>
-        public float NumericValue { get { return this.value; } }
+        public float NumericValue => value;
 
         /// <summary>
         /// Returns the angular value of the angular velocity.
         /// </summary>
-        public Angle AngleValue { get { return Angle.FromRadians(this.value); } }
+        public Angle AngleValue => Angle.FromRadians(value);
 
         /// <summary>
         /// Returns an angular velocity with value 0.
         /// </summary>
-        public static AngularVelocity Zero { get { return new AngularVelocity(0); } }
+        public static AngularVelocity Zero => new AngularVelocity(0);
 
         #endregion
 
         #region methods
 
         #region equality and hashcode
 
-        public bool Equals(AngularVelocity other)
-        {
-            return this.value == other.value;
-        }
+        // ReSharper disable once CompareOfFloatsByEqualityOperator
+        public bool Equals(AngularVelocity other) => value == other.value;
```

Comment:
That comparison basically does the same, for all values absolutely larger than ~1, and it will be much too insensitive for very small (absolutely) numbers.

More info: https://randomascii.wordpress.com/2012/02/25/comparing-floating-point-numbers-2012-edition/

TL;DR: Never use float quality after calculations. There are ways to fake it, but they aren't perfect. But that's besides the points. You should simply not use it for that.
What you can use it for us equality after assignment, which holds. This could be important if using the type as a key or similar.
In other words, I argue this implementation is correct.